packages/system/systemd/build.yaml

```diff
@@ -9,19 +9,32 @@ prelude:
+  # To get systemd-measure to build we need to set blkid, openssl and bootloader to true
+  # Set also the shared lib name with an appended kairos so we can use it everywhere without overwriting the system one
+  - PACKAGE_VERSION=${PACKAGE_VERSION%\+*} && cd src/systemd && meson setup build -Dmode=release -Dbootloader=true -Defi=true -Dblkid=true -Dopenssl=true -Dshared-lib-tag=${PACKAGE_VERSION}-kairos -Dnss-resolve=disabled -Dlogind=false -Dcoredump=false -Dhomed=disabled -Dfirstboot=false -Dhostnamed=false -Dhibernate=false -Dinitrd=false -Dimportd=false -Dkernel-install=false -Dlocaled=false -Dmachined=false -Dnetworkd=false -Dnss-myhostname=false -Dnss-mymachines=false -Dnss-systemd=false -Doomd=false -Dportabled=false -Dhwdb=false -Dpstore=false -Dquotacheck=false -Drandomseed=false -Drepart=false -Dresolve=false -Drfkill=false -Dsysext=false -Danalyze=false -Dsysupdate=false -Dsysusers=false -Dstoragetm=false -Dtimedated=false -Dtimesyncd=false -Dtmpfiles=false -Duserdb=false -Dvconsole=false -Dxdg-autostart=false -Didn=false -Dpolkit=false -Dnscd=false -Dkmod=false -Ddbus=false -Dglib=false -Dbacklight=false -Dldconfig=false -Dgshadow=false -Dwheel-group=false -Dadm-group=false -Dxkbcommon=false -Dzstd=false -Dlz4=false -Dutmp=false -Dlink-udev-shared=false -Dlink-systemctl-shared=false -Dlink-networkd-shared=false -Dlink-timesyncd-shared=false -Dlink-journalctl-shared=false -Dlink-boot-shared=false -Dlink-portabled-shared=false -Denvironment-d=false -Dqrencode=false -Dpwquality=false -Dlibcurl=false -Dfdisk=false -Dlibidn2=false -Dlibiptc=false -Ddns-over-tls=false -Didn=false -Dgnutls=false -Dp11kit=false -Dlibidn=false -Dlibidn2=false -Dgcrypt=false -Dxz=false -Dzlib=false -Dbzip2=false
```

Comment:
no clue what changed here :/

Reply:
its on the comment above, to make meson build the systemd-measure binary we need to enable:
 - openssl (-Dopenssl=true)
 - blkid (-Dblkid=true)
 - bootloader, was already enable to build the efi artifacts (-Dbootloader=true)

Then to generate a custom shared lib with a kairos name on it so it doesnt collide with installed libs we set `-Dshared-lib-tag=${PACKAGE_VERSION}-kairos` which will generate a `libsystemd-shared-255-kairos.so` which the systemd-measure binary will look for

I moved all those options to be at the start of the command so they are "easy" to check xD

Reply:
gotcha, thanks for clarifying